backend/app/domain/comment.go

```diff
@@ -0,0 +1,35 @@
+package domain
+
+import "time"
+
+type Comment struct {
+	threadKey string
+	commentKey string
+	contributer string
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`contributer` is a misspelling of `contributor` (misspell)